metrics/issues-open.md

```diff
@@ -1,54 +1,55 @@
-# Open Issues

```

Comment:
I think the text in here is applicable to the Issues-Active metric. We shouldn't lose this work.

Reply:
We have Open Issues somewhere else, I think...

Active issues are issues active (having some activity, such as comments, changes in state) during a certain period.

Open issues are issues open at a certain point in time.

So, both metrics are different.... But as I said, I guess we still have the text for open issues elsewhere.

Reply:
OK, just clarify in this respect, since we have three different merics that could be mistaken (of of them mentioned in the code development focus area):

* Issues: Number of new issues (opened during a certain period)
* Issues_Active: Number of issues that showed some activity (during a certain period)
* Issue_Backlog: How many issues are still open? (at a given point in time)

This patch deals with the first two. In particular, Issues, as you mention below, could also be Issues_Opened, or also, New_Issues. The third one is "issues that remain open right now (or at any other given point in time), and from that point of view, could also be named "Issues_Open", but that could be confusing with "Issues_Opened". This is the reason why I was trying to avoid "open" and "opened" in the metric name.

Reply:
And in any case, @sgoggins wrt to your original message, i agree: `issues-open.md` in this change is still in the repository, and I plan to reuse it for "Issues_Backlog" (see the file for the code development focus area).

Reply:
New issues normally are associated to a time frame namely in the last x hours, days or that have not been acknowledged (In a monitoring system). Thus I would propose to pick a time frame, although there will be the caveat that some issues may be resolved within the chosen time frame and technically would no longer be new issues.
--------------------
Metrics 2 and 3 could be misinterpreted since Active and Backlog are interchangeable terms in an operational environment but in an Agile environment "represents an option the team has for delivering a specific outcome rather than a commitment".
 
Pending, unresolved, fixed and closed are more finite status indicators that I think most understand.
--------------------
Two important metrics that I personally find of value are Issue_type and Issue_severity which help prioritization and picture the response to the respective categories or combinations.